components/launcher-client/src/client.rs

```diff
@@ -21,21 +21,24 @@ use ipc_channel::ipc::{IpcOneShotServer,
                        IpcReceiver,
                        IpcSender};
 use std::{collections::HashMap,
-          fs,
           io,
           path::Path};
 
 type Env = HashMap<String, String>;
 type IpcServer = IpcOneShotServer<Vec<u8>>;
 
 pub struct LauncherCli {
-    tx:   IpcSender<Vec<u8>>,
-    rx:   IpcReceiver<Vec<u8>>,
+    tx: IpcSender<Vec<u8>>,
+    rx: IpcReceiver<Vec<u8>>,
+    #[cfg_attr(windows, allow(dead_code))]
```

Comment:
In this case the field is still necessary on windows, it's just not *assigned* to, so it triggers the `dead_code` warning, is that right? A comment to explain that subtlety may be worthwhile.